### DISCUSSION TITLE
A prefix activating WebChatGPT

### DISCUSSION THREAD

**Discussion by kpion** [qunash/chatgpt-advanced#136] [Ideas]
Hi.

Right now to enable / disable WebChatGPT one needs to use a mouse (alt+w doesn't work in my case,  Kubuntu+Firefox).

It would be cool if one could **prefix** a query with something, to activate the extension. Like:
`@interesting facts about Penguins`: the extension would do its job. But:
`interesting facts about Penguins`: the extension would do exactly nothing. So we'd get a standard ChatGPT answer.

Thanks for the extension btw, pretty smart :)

**Comment by Neelanjan-chakraborty** [comment]


**Reply by kpion** [reply]
OK, I'll try this one too, thanks
